Eastern-Research-Group/mywaterway
# Feature/refactor classes to hooks

## Related Issues:
* https://app.breeze.pm/projects/100762/cards/3330906

## Main Changes:
* The last 3 class-based Community tabs (Restore, Identified Issues, and the Monitoring tab) are now refactored to use React Hooks. 

## Steps To Test:

Tabs to check:
**Monitoring
Restore
Identified Issues**

1. Navigate to a few testing locations on the Community page and try the above tabs to verify there are no regressions or new issues introduced.
2. Change locations while on the tab and check that new data populates correctly, toggle all switches, and check the Monitoring tab sorts.

Note that the Monitoring and Identified Issues tabs perform noticeably slower while Chrome devtools are opened. This issue has been in the app for a long time and can be reproduced on the dev and staging sites. As our users would not see this and have not complained so far I don't think it's an issue.
